advanced_counter/apps/advanced_counter/lib/advanced_counter/repos/cockroach.ex

```diff
@@ -0,0 +1,5 @@
+defmodule AdvancedCounter.Repos.Cockroach do
+  use Ecto.Repo,
+    otp_app: :advanced_counter,
+    adapter: Ecto.Adapters.Postgres
```

Comment:
Just to sanity check, does this work in prod? I've had issues with Ecto and Cockroach before. Are those solved and the default postgres adapter works nowadays, or do you need https://hexdocs.pm/ecto_cockroachdb?

Reply:
@thruflo it does work completely, as stated in the header of the repo of that package. The only thing is that I needed to disable the `migration_lock` for cockroach